src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/UpdateEnglishFractions/UpdateEnglishFractionsCommandHandler.cs

```diff
@@ -72,7 +63,7 @@ public class UpdateEnglishFractionsCommandHandler : AsyncRequestHandler<UpdateEn
                             new DasEnglishFraction
                             {
                                 EmpRef = fractionCalculations.Empref,
-                                DateCalculated = DateTime.Parse(calculations.CalculatedAt),
+                                DateCalculated = calculations.CalculatedAt,
                                 Amount = decimal.Parse(fraction.Value)
```

Comment:
This is using the locale of the current thread to parse the string. Can we rely on this always being correct or is the string always in the UK format?

Reply:
Not sure on this one, the old code was doing string parsing, this is now a DateTime to DateTime!

Reply:
Ah yes, so it is.